hack/lib/etcd.sh

```diff
@@ -16,10 +16,14 @@
 
 # A set of helpers for starting/running etcd for tests
 
-ETCD_VERSION=${ETCD_VERSION:-3.0.4}
+ETCD_VERSION_DEFAULT=3.0.4
+ETCD_VERSION=${ETCD_VERSION:-$ETCD_VERSION_DEFAULT}
 ETCD_HOST=${ETCD_HOST:-127.0.0.1}
 ETCD_PORT=${ETCD_PORT:-2379}
 
+# This is the SHA256 digest of the linux-amd64 release tarball.
```

Comment:
This is definitely my least favorite part. Do you have any ideas on how to make this less gross, or better commented? I tried putting it in an associative array (keyed off the ETCD version), but that doesn't work on OSX bash, and it just got worse.

The digest is used to ensure that external resources are what we expect.